custom-pages/case-autogenerated-overview/gulpfile.js

```diff
@@ -0,0 +1,217 @@
+/* jshint node:true */
+
+'use strict';
+
+var gulp = require('gulp');
+var path = require('path');
+var properties = require('properties');
+var fs = require('fs');
+var plumber = require('gulp-plumber');
+
+/* server */
+var connect = require('gulp-connect');
+var rewriter = require('connect-modrewrite');
+var launch = require('gulp-open');
+
+/* build */
+var usemin = require('gulp-usemin');
+var htmlmin = require('gulp-htmlmin');
+var rev = require('gulp-rev');
+var clean = require('gulp-clean');
+var html2js = require('gulp-ng-html2js');
+var concat = require('gulp-concat');
+var zip = require('gulp-zip');
+var runSequence = require('run-sequence');
+
+/* javascript */
+var uglify = require('gulp-uglify');
+var jshint = require('gulp-jshint');
+
+/* test */
+var gulp = require('gulp');
+var karma = require('karma').server;
+var protractor = require('gulp-protractor').protractor;
+
+/**
+ * Configuration
+ */
+var pageContent = fs.readFileSync('src/page.properties', 'utf8');
+var customPageName = properties.parse(pageContent).name;
+var folderName = path.basename(__dirname);
+
+var opt = {
+  port: 3000,
+  livereload: 31357
+};
+
+var htmlminOpt = {
+  collapseWhitespace: true,
+  removeComments: true,
+  useShortDoctype: true
+};
+
+var useminOpt = {
+  css: ['concat', rev()],
+  html: [htmlmin(htmlminOpt)],
+  js: [uglify(), rev()]
+};
+
+function serve(configuration) {
+  return connect.server({
+    root: ['src', '.'],
+    port: opt.port,
+    livereload: configuration.livereload,
+    middleware: function () {
+      return [
+        rewriter([
+          '^/bonita/portal http://localhost:8080/bonita/portal [P]',
```

Comment:
This is used in local when developping. There is no impact when embedded in the product.